grid/pdf-export-server/PdfExport/PdfExporter.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Telerik.DataSource;
+
+namespace PdfExport
+{
+    public class PdfExporter
+    {
+        public async Task<byte[]> ExportWithRadPdfProcessing<T>(IQueryable<T> data, DataSourceRequest gridRequest)
+        {
+            var pdfExporter = new RadPdfProcessingExporter();
+
+            return await pdfExporter.ExportPdf(data, gridRequest);
```

Comment:
The `gridRequest` variable can be renamed to `dataSourceRequest` as the data is not restricted to a specific grid form - the functionality can be used with a sample datasource regardless of having a grid or not.